src/childPaths.js

```diff
@@ -17,7 +17,7 @@ export default function childPaths (node) {
 		return [];
 	}
 
-	const childProperties = context.getProperties(node);
+	const childProperties = context.getChildProperties(node);
```

Comment:
This was a bug

Reply:
These kinds of things don't need to be part of PRs, you can commit them directly